game/lib/game/game_runner.py

```diff
@@ -149,3 +184,28 @@ def get_main_hero(self):
 
     def get_game(self):
         return self.__game
+
+    def selected_weapon(self, horizontal, vertical):
+        marked_index = self.__main_hero.get_equipment().get_marked_index()
+        marked_item = self.__main_hero.get_equipment().get_item_by_index(marked_index)
+        distance = 1
```

Comment:
melee i distance jest zapisane w pliku konfiguracyjnym. Skoro one nie są edytowane w tej metodzie to wydajniej je wrzucić do klasy, wpisać im wartość z pliku konfiguracyjnego i tylko tutaj używać, zamiast tworzyć nową instancję z każdym wywołaniem metody